### ISSUE TITLE
mue conversion too small?

### ISSUE THREAD

**Issue by xmarcano** [flav-io/flavio#126]
Dear Flavio team, 

We are trying to compute several LFV observables, however the numbers we are getting for mu-e conversion in nuclei seem too small. For example, 

```
wc  = Wilson({'phil1_12': 1e-10},scale=1e3, eft='SMEFT', basis='Warsaw')
flavio.np_prediction('BR(mu->eee)',wc)
flavio.np_prediction('CR(mu->e, Al)',wc)

5.912821433496535e-12
1.1867886618408577e-19
```

Nevertheless, from using wilson for the RGEs and plugging the WCs at low energies in available expressions (e.g. 1702.03020), we expected CR~6e-11.

Trying to understand the difference, we checked the mueconversion.py file and found some apparent inconsistency when defining the gRV and similar couplings (line 30 on). Reference hep-ph/0203110 includes a normalization of Gf/sqrt(2) for the 2l2q operators, however the WET.flavio.pdf document doesn't (it does for flavor conserving ones, but not for the LFV ones). Therefore, we understand that there are some factors missing in mueconversion.py and one should replace

```
30    gRV = {'u': 4*(wc['CVRR_mueuu'] + wc['CVLR_mueuu']),...

by 

30    gRV = {'u': 1/(np.sqrt(2)*par['GF'])*(wc['CVRR_mueuu'] + wc['CVLR_mueuu']),...
```
and so on for gLV... This new normalization enhances flavio predicition for CR up to 3e-11, consistent with what we were expecting. 

Does this make sense? is it possible that this factor is missing in the code? or is the normalization in WET.flavio.pdf what needs to be modified?

Many thanks in advance!
Best regards, 
Xabi

**Comment by peterstangl** [comment]
@xmarcano thank you for reporting this issue. I have taken a look at the implementation and I think you are right and a wrong normalization is used there for the `gRV`, `gLV`, `gRS`, and `gLS` coefficients. On the other hand, the normalization of `AL` and `AR` seems to be fine.

Since you have already suggested a possible fix, would you like to open a pull request and implement this modification?

Apart from this, I also noticed that the `GuS` and `GdS` factors seem to be used in an inconsistent way. According to hep-ph/0203110 (bottom of page 6), `GuS`_proton = `GdS`_neutron and `GdS`_proton = `GuS`_neutron. However, in
https://github.com/flav-io/flavio/blob/8efb73d238d956f8717abe50e1fa9a6e03edb5f6/flavio/physics/mudecays/mueconversion.py#L44-L45
and
https://github.com/flav-io/flavio/blob/8efb73d238d956f8717abe50e1fa9a6e03edb5f6/flavio/physics/mudecays/mueconversion.py#L48-L49
the same factors related to u and d quarks are used for both proton and neutron, which also seems to be wrong and needs to be fixed.

**Comment by peterstangl** [comment]
fixed by PR #132
